lib/tools/apk-signing.js

```diff
@@ -152,11 +151,14 @@ apkSigningMethods.checkApkKeystoreMatch = async function (keytool, md5re, keysto
     log.debug(`keystore MD5: ${keystoreHash}`);
     let matchesKeystore = entryHash && entryHash === keystoreHash;
     log.debug(`Matches keystore? ${matchesKeystore}`);
+
+    // If we have a keystore match, stop iterating
     if (matchesKeystore) {
-      return true;
+      foundKeystoreMatch = true;
+      return false;
```

Comment:
false flag does not look very obvious IMHO. It would be more readable to have an explicit argument for  such purpose like in objectiveC blocks:

```objc
[array enumerateObjectsUsingBlock:^(id obj, NSUInteger idx, BOOL *stop) {
    //…
    if(idx == 1000) 
        *stop = YES;
}];
```